docs/ref/models/instances.txt

```diff
@@ -309,6 +309,21 @@ pass a dictionary mapping field names to errors::
         'pub_date': ValidationError(_('Invalid date.'), code='invalid'),
     })
 
+.. note::
```

Comment:
Generally I prefer to use `.. admonition:: descriptive title goes here` rather than `.. note::` to make the docs a bit more skimmable.